docs/pages/database-access/guides/azure-sql-server-ad.mdx

```diff
@@ -0,0 +1,183 @@
+---
+title: Database Access with Azure SQL Server with Active Directory authentication (Preview)
+description: How to configure Teleport Database Access with Azure SQL Server with Active Directory authentication.
```

Comment:
```suggestion
title: Database Access with Azure SQL Server with Azure Active Directory authentication (Preview)
description: How to configure Teleport Database Access with Azure SQL Server with Azure Active Directory authentication.
```
nit: suggest using "Azure Active Directory" to be more descriptive, same as my suggestion below